app/test_main.py

```diff
@@ -1,3 +1,43 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_list_with_two_values() -> None:
+    assert (
+        isinstance(get_human_age(1, 1), list)
+        and len(get_human_age(1, 1)) == 2
+    ), f"must return list but was {type(get_human_age(1, 1))}"
+
+
+def test_should_return_integers_numbers_in_list() -> None:
+    assert (
```

Comment:
your test data in below test checks that result is a list of integers - so you don't need this test really